### PR TITLE
Upgrade Alt-1 to Rocky release

### PR DESCRIPTION
**Pull request by dougszumski**

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ kolla_ansible_source_url: https://github.com/stackhpc/kolla-ansible.git
 
 # Version (branch, tag, etc.) of Kolla Ansible source code repository if type
 # is 'source'.
-kolla_ansible_source_version: refs/tags/stackhpc/6.1.0.6
+kolla_ansible_source_version: stackhpc/rocky
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml]: Can't really merge with this in place until we upgrade the rest.

Comment (dougszumski) [reply etc/kayobe/kolla.yml]: Yeah, i'm just working off this branch for now and will use the relevant tags for the upgrade.

File: etc/kayobe/kolla/globals.yml [hunk 2]
```diff
@@ -19,29 +19,41 @@ monasca_default_authorized_roles:
...
+elasticsearch_tag: 7.1.0.1
+fluentd_tag: 7.1.0.1
+glance_tag: 7.0.2.1-1
+haproxy_tag: 7.0.2.1-1
+heat_tag: 7.0.2.1-1
+horizon_tag: 7.0.2.1-1
+influxdb_tag: 7.1.0.1
+ironic_tag: 7.0.2.1-1
+iscsid_tag: 7.0.2.1-1
+kafka_tag: 7.1.0.1
```
Comment (dougszumski) [inline etc/kayobe/kolla/globals.yml:43]: Now I noticed I named the tag incorrectly - thanks for the tool.

**Comment by dougszumski** [comment]
+1 from me, but I can't approve your changes since I created the PR.

**Review by markgoddard — APPROVED** [review]
Upgrade went well overall, now seems to be working with these changes.

**Pull request merged**